src/erlcloud_ec2_meta.erl

```diff
@@ -25,12 +26,13 @@ get_instance_metadata(Config) ->
 %% @doc Retrieve the instance meta data for the instance this code is running on. Will fail if not an EC2 instance.
 %%
 %% This convenience function will retrieve the instance id from the AWS metadata available at 
-%% http://169.254.169.254/latest/meta-data/*
+%% http://<host:port>/latest/meta-data/*
```

Comment:
Should we document how to override this, at least the label of the application configuration parameter (`metadata_host_port`)? Also, would it be better for the label to be `ec2_meta_host_port`?

Reply:
The first question is what I already asked in the description, basically: "Would you like this option to be exposed somewhere (e.g. doc.)?", so it might not be for me (not sure).

As for the label, I don't mind changing it; it's fairly innocuous. I let you decide.

Reply:
ec2_meta_host_port is preferable and as i mentioned in other comment - app.src is a good place for doc s for this.

Reply:
6a337be